hr_timesheet_begin_end_no_end/tests/test_timesheet_begin_end_no_end.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import exceptions, fields
+from odoo.tests.common import TransactionCase
+
+
+class TestBeginEndNoEnd(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.analytic = cls.env.ref("analytic.analytic_administratif")
+        cls.user = cls.env.ref("base.user_root")
+
+    def base_line(self):
+        return {
+            "name": "test",
+            "date": fields.Date.today(),
+            "time_start": 10.0,
+            "time_stop": 12.0,
+            "user_id": self.user.id,
+            "account_id": self.analytic.id,
+            "amount": -60.0,
```

Comment:
the demo data should reflect timesheet lines data, thus with a `project_id` and an `employee_id`. `user_id`, `account_id` and `amount` are not needed in this case.

Reply:
hi @huguesdk, i am not certain i agree.

- in `hr_timesheet_begin_end`, the test data is very similar to the above. if we change it here, we should change it there.
- `project_id` and `employee_id` are not needed by any of the code being tested.

you're _probably_ right that your proposed test data more closely matches the intended use-case, but i do not know whether the test data should be minimally viable or whether it should match expected circumstances. i err towards the first.

Reply:
on second reading, i agree with you. `hr_timesheet` uses project ids and employees. i will change this.